app/views/users/show.html.erb

```diff
@@ -1 +1,36 @@
-<p>Here is a user with this id</p>
+<section id='user_box'>
+  <img class='image' src=<%= @user.photo %> alt='profile picture' />
+  <div class='user_name'>
+    <h1 class='name'><%= @user.name %></h1>
+    <p class='posts_counter'>Number of posts: <%= @user.posts_counter%></p>
+  </div>
+</section>
+<section class='bio'>
+  <h4>Bio</h4>
+  <p><%= @user.bio %></p>
+</section>
+
+<section id='posts'>
+  <% @posts.each_with_index do |p, index| %>
```

Comment:
- [ ] Please use the `three_most_recent_posts` method you created in your users model to display the recent posts and not all the posts 👍